SwarmHeader header
FlightStatus flightStatus
StateStatus stateStatus


